Spelling, grammar, and link check (#95)
conveyal/docs: docs/prepare-data/derivations.mdx

* Polygon Shapefile → Aggregation Area 

## Aggregation Areas

> Aggregation areas are used to [summarize regional analysis results](/analysis/regional#measuring-aggregate-accessibility). To create an aggregation area, use the [Data Sources](/prepare-data) panel to upload and process a polygon shapefile or polygon GeoJSON. Support for additional spatial formats (e.g. geopackage files) may be added in future releases.

Reviewer: For now, aggregation areas can only be created from shapefiles.

```suggestion
Aggregation areas are used to [summarize regional analysis results](/analysis/regional#measuring-aggregate-accessibility). To create an aggregation area, use the [Data Sources](/prepare-data) panel to upload and process a polygon shapefile. Options to create aggregation areas from other spatial data source formats (e.g. geopackage and GeoJSON files) may be added in future releases.
```
